taskcluster/docker/base/Dockerfile

```diff
@@ -52,6 +52,30 @@ RUN pip install 'taskcluster>=4,<5'
 
 RUN locale-gen en_US.UTF-8
 
+# TODO Extract the test tools in a separate Dockerfile once the UI tests fetches APKs from build tasks
```

Comment:
You could have a new docker image that depends on this image (with `FROM ${DOCKER_IMAGE_PARENT}` and adding a `parent: base` to the stanza in `taskcluster/ci/docker-image/kind.yml`.

Reply:
TIL! Thanks for pointing it out. Done!